pkg/oc/bootstrap/docker/run_self_hosted.go

```diff
@@ -282,7 +334,7 @@ func (c *ClusterUpConfig) makeMasterConfig() (string, error) {
 	container.MasterImage = c.openshiftImage()
 	container.Args = []string{
 		"--write-config=/var/lib/origin/openshift.local.config",
-		"--master=127.0.0.1",
+		fmt.Sprintf("--master=%s", c.ServerIP),
```

Comment:
@deads2k not sure if this fixes the comment above.

Reply:
> @deads2k not sure if this fixes the comment above.

You had to change this because you needed to serve from a different nic